docs/src/Avatar.doc.js

```diff
@@ -27,6 +27,12 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'outline',
+        type: 'boolean',
+        defaultValue: false,
+        description: 'Adds a white border around Avatar',
```

Comment:
Can we maybe add why you'd do this? "So it's visible when displayed on other images"

Reply:
@chrislloyd definitely, added.